code/core/src/manager/components/sidebar/Tree.tsx

```diff
@@ -472,7 +479,7 @@ const Root = React.memo<NodeProps & { expandableDescendants: string[] }>(functio
 export const Tree = React.memo<{
   isBrowsing: boolean;
   isMain: boolean;
-  status?: State['status'];
+  allStatuses?: StatusesByStoryIdAndTypeId;
```

Comment:
Why is this `allStatuses` and not just `statuses`? It makes me think there must be a subset of statuses as well somewhere.

Reply:
well there could be subset of statuses, like statuses for a given story, or a given type.

This is explicit in saying "all the statuses in the system".